src/igvfd/upgrade/open_reading_frame.py

```diff
@@ -1,6 +1,19 @@
 from snovault import upgrade_step
 
 
+@upgrade_step('open_reading_frame', '2', '3')
+def open_reading_frame_2_3(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-1792
+    notes = value.get('notes', '')
+    if 'genes' not in value and 'gene' in value:
+        value['genes'] = value.pop('gene', None)
+        if 'notes' in value:
+            value['notes'] = f'{notes}. This object used gene as a property, which is now upgraded to genes.'
+        else:
+            value['notes'] = 'This object used gene as a property, which is now upgraded to genes.'
+    return
```

Comment:
Later upgrades should go below earlier upgrades.
The if statement also doesn't need to check for if 'genes' not in value.

Reply:
Step moved and removed the `genes` check